docs/documentation/platform/event-subscriptions.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: "Event Subscriptions"
+sidebarTitle: "Events"
+description: "Subscribe to events in Infisical in real-time"
+---
+
+<Info>
+<strong>Note:</strong> Event Subscriptions is a paid feature.<br /><br />
+On Infisical Cloud, this feature is available on the <strong>Pro</strong> and <strong>Enterprise</strong> tiers, each providing different event retention periods.<br />
+For self-hosted deployments, please contact <a href="mailto:sales@infisical.com">sales@infisical.com</a> to purchase an enterprise license.
+</Info>
+
+## Introduction
+
+Event Subscriptions in Infisical let you receive real-time notifications when specific actions take place within your account or organization. These can include changes to secrets, users, teams, and other important resources.
+
+Currently, you can subscribe to notifications for the following event types:
+
+### Supported Event Types
+
+- `secret:created`: Triggered when a secret is created.
+- `secret:updated`: Triggered when a secret is updated.
+- `secret:deleted`: Triggered when a secret is deleted.
+
+## Permissions Setup
+
+Proper permissions are required for configuring event subscriptions. To set up the necessary permissions, follow these steps:
```

Comment:
we should try to explain at a high level how permission works with events. something like: To listen to events, you need to have the permission on the resource you intend to listen to. Below is an example that ...